src/components/ProductMilestoneDetailPage/ProductMilestoneDetailPage.tsx

```diff
@@ -1,9 +1,68 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { useServiceContainer } from 'hooks/useServiceContainer';
+
+import { AttributesItems } from 'components/AttributesItems/AttributesItems';
 import { ContentBox } from 'components/ContentBox/ContentBox';
+import { OptionalText } from 'components/OptionalText/OptionalText';
+import { ProductMilestoneCloseStatusLabel } from 'components/ProductMilestoneCloseStatusLabel/ProductMilestoneCloseStatusLabel';
 import { useServiceContainerMilestone } from 'components/ProductMilestonePages/ProductMilestonePages';
+import { ServiceContainerLoading } from 'components/ServiceContainers/ServiceContainerLoading';
+
+import * as productMilestoneApi from 'services/productMilestoneApi';
+
+import { createDateTime } from 'utils/utils';
 
 export const ProductMilestoneDetailPage = () => {
+  const { productMilestoneId } = useParams();
+
   const { serviceContainerMilestone } = useServiceContainerMilestone();
-  console.log('TEMPORARY:', serviceContainerMilestone);
 
-  return <ContentBox padding marginBottom></ContentBox>;
+  const serviceContainerCloseResults = useServiceContainer(productMilestoneApi.getCloseResults);
+  const serviceContainerCloseResultsRunner = serviceContainerCloseResults.run;
+
+  useEffect(() => {
+    serviceContainerCloseResultsRunner({
+      serviceData: { id: productMilestoneId },
+      requestConfig: { params: { latest: 'true' } },
+    });
+  }, [serviceContainerCloseResultsRunner, productMilestoneId]);
+
+  const attributes = [
+    {
+      name: 'Status',
+      value: serviceContainerMilestone.data?.endDate ? 'CLOSED' : 'OPEN',
+    },
+    {
+      name: 'Start Date',
+      value: (
+        <OptionalText>{createDateTime({ date: serviceContainerMilestone.data.startingDate, includeTime: false })}</OptionalText>
```

Comment:
The date format is different from our angularjs version, are we going to refactor them in a new way or we should keep it as the same one with before?
![image](https://user-images.githubusercontent.com/6870455/221869233-a837885b-2e6a-4a9d-9f32-805115c315b9.png)
![image](https://user-images.githubusercontent.com/6870455/221869289-ec199ce5-f57b-4bc8-bfce-a26ca2746b81.png)

@matedo1

Reply:
This format is used in **createDateTime** function so I think it is correct.